src/main/resources/application.properties

```diff
@@ -1,12 +1,19 @@
-spring.application.name=eiffel-intelligence
+spring.application.name: eiffel-intelligence
 
 # An extra optional version tag, version will be added after GitHub release version.
 # Example if build.version=1.2.3-internal becomes Version 0.0.18 (1.2.3-internal)
-build.version=
+build.version:
```

Comment:
Is it this build version they talk about in https://github.com/eiffel-community/eiffel-intelligence/issues/252

Reply:
Could be, I was actually thinking about why it was even there while I was working on this story. I wasn't aware that there was an issue about this. I can remove it since I don't believe we've used this property.

Reply:
Yes, that is that property that issue refer to.
I believe its enough with the version that comes from the pom file.

Reply:
I'll remove it then.